File: Estrutura_Sequencial/Clojure/Ex1019.clj
(def tempo (read))

(def Horas (/ tempo 3600))
(def H (mod Horas 3600))

(def Minutos (/ tempo 60))
(def M (mod Minutos 60))

(def Segundos (mod tempo 60))

(printf "%d:%d:%d\n" (int H) (int M) (int Segundos))

File: Estrutura_Sequencial/C#/Ex1019.cs
using System;

class MainClass {
    public static void Main (string[] args) {
        int totalSegundos = Convert.ToInt32(Console.ReadLine());

        int horas = totalSegundos / 3600;
        int minutos = (totalSegundos % 3600) / 60;
        int segundos = totalSegundos % 60;

        Console.WriteLine($"{horas}:{minutos}:{segundos}");
    }
}